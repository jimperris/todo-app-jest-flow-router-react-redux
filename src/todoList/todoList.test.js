import TodoList from './todoList.jsx'
import { shallow } from 'enzyme'
import React from 'react'
describe('post', () => {
  const todos = [{title: 'jazzo', completed: false, id: 0}, {title: 'zoo', completed: true, id: 1}]
  it('should render and keep props', () => {
    const wrap = shallow(
      <TodoList todos={todos} />
    )
    expect(wrap.exists()).toBeTruthy()
    expect(wrap).toMatchSnapshot()
  })
})
