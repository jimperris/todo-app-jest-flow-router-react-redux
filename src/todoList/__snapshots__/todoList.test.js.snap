// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`post should render and keep props 1`] = `
<div>
  <div
    key="0"
  >
    <Link
      replace={false}
      to="/todos/0"
    >
      jazzo
    </Link>
  </div>
  <div
    key="1"
  >
    <Link
      replace={false}
      to="/todos/1"
    >
      zoo
    </Link>
  </div>
</div>
`;
